src/decorators.ts

```diff
@@ -490,12 +491,11 @@ export function Workflow(config: WorkflowConfig={}) {
   return decorator;
 }
 
-export function Transaction(config: TransactionConfig={}) {
+export function Transaction<T extends UserDatabaseClient>(config: TransactionConfig={}) {
```

Comment:
We're currently doing some refactoring on Transaction Context. So I'd remove this change to `Transaction` and imports from `UserDatabaseClient`.